src/assets/scss/layout/layout.scss

```diff
@@ -0,0 +1,50 @@
+@import "../../../assets/scss/helpers/variables";
+
+.pf-c-dropdown {
+  &__menu {
+    &-item {
+      color: #000 !important;
+    }
+  }
+}
+
+.footer {
+  display: flex;
+  position: fixed;
```

Comment:
Don't remember what the final decision was for the positioning of the footer, but if we wanted the footer to stick to the bottom of the viewport and not overlap the contents of the page, we can set `position: absolute` here and set the `height: 100vh` for the Page component inside PageWrapper.

Reply:
Is there any reason to use position `absolute` over `fixed`?